scripts/segmentation/test.py

```diff
@@ -194,6 +194,7 @@ def benchmarking(model, args):
 
 if __name__ == "__main__":
     args = parse_args()
+    args.test_batch_size = max(1, args.ngpus)
```

Comment:
Just want to know why `args.test_batch_size` here is dependent on `args.ngpus`. Also since `args.batch_size` is already defined, is it possible to use this one directly instead of adding a new one like `test_batch_size`?

Reply:
Thank you for the review! I have removed this line, and I will update the code later.
The `test_batch_size` is defined in [`train.py`](https://github.com/dmlc/gluon-cv/blob/master/scripts/segmentation/train.py#L51), and it is the batch size of testing dataset. For testing script, we should keep the consistency with training procedure.

Reply:
Thank you for the review! I have removed this line, and I will update the code later.
The `test_batch_size` is defined in [`train.py`](https://github.com/dmlc/gluon-cv/blob/master/scripts/segmentation/train.py#L51), and it is the batch size of testing dataset. For testing script, we should keep the consistency with training procedure.

Reply:
I don't know why the program gets stuck. I will check it later.